composer.json

```diff
@@ -21,7 +21,7 @@
     "extra": {
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "php": "^7.3 || ~8.0.0 || ~8.1.0 || ~8.2.0",
```

Comment:
We should drop PHP 7

Reply:
`config.platform` needs to pin the PHP version

Reply:
done